azext_iot/tests/iothub/test_iot_ext_int.py

```diff
@@ -32,10 +33,28 @@
 
 CWD = os.path.dirname(os.path.abspath(__file__))
 
+test_managed_name = "TESTMANAGEDIDENTITY"
```

Comment:
Could generate a unique name ? We use `azext_iot.tests.generators.generate_generic_id` across tests for this purpose.